pkg/trace/config/config.go

```diff
@@ -351,6 +351,10 @@ type AgentConfig struct {
 	ConnectionLimit int    // for rate-limiting, how many unique connections to allow in a lease period (30s)
 	ReceiverTimeout int
 	MaxRequestBytes int64 // specifies the maximum allowed request size for incoming trace payloads
+	TraceBuffer     int   // specifies the number of traces to buffer before blocking.
+	Decoders        int   // specifies the number of traces that can be concurrently decoded.
```

Comment:
I believe it's the number of concurrent decoding workers, not traces.

Reply:
I think that's the same thing, but I will clarify.

Reply:
Couldn't find a better wording lol, let's keep it decoders 🤷